assets/sass/cds/_sections.scss

```diff
@@ -645,6 +646,8 @@ span.livefr {
         width: 65%;
       }
     }
+  }
+}
```

Comment:
This moves all the following styles out from under `.work-in-progress--product-module` and into the top level. Is that what we want? The indenting seems to imply they're part of `.work-in-progress--product-module` ?

Reply:
Yes, it looks like a whole bunch of styles that didn't belong there, were moved under `.work-in-progress--product-module`, so certain styles couldn't pick up the css because they did not have the right class

Reply:
Do we know if this fixes all of them, looking at #2394 there is a huge merge in sections which looks like it does a lot of reformatting? Shoudl we revert #2394 and then reapply it as a formatting step and then the change so we can see what's going on?